monix-eval/shared/src/test/scala/monix/eval/TaskCancelableSuite.scala

```diff
@@ -140,92 +140,97 @@ object TaskCancelableSuite extends BaseTestSuite {
     assert(sc.state.tasks.isEmpty, "tasks.isEmpty")
   }
 
-  test("Task.cancelable0(allowContinueOnCallingThread = true) should be stack safe on repeated, right-associated binds") {
-    implicit s =>
-      def signal[A](a: A): Task[A] =
-        Task.cancelable0[A]({ (_, cb) =>
-          cb.onSuccess(a)
-          Task.unit
-        }, allowContinueOnCallingThread = true)
+  test("Task.cancelable0() should be stack safe on repeated, right-associated binds") { implicit s =>
```

Comment:
I think all of the tests that used to be with `allowContinueOnCallingThread = true` are duplicated in this file now so we could remove them / revert to the state from before my change